src/main/java/Jeff.java

```diff
@@ -0,0 +1,91 @@
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Jeff {
+    public static class Task {
+        private String desc;
+        private boolean done;
+        public Task(String desc) {
+            this.desc = desc;
+            this.done = false;
+        }
+
+        public void setDone() {
+            this.done = true;
+        }
+
+        public void setNotDone() {
+            this.done = false;
+        }
+
+        @Override
+        public String toString() {
+            if (this.done) {
+                return "[X] " + this.desc;
+            }
+            return "[ ] " + this.desc;
+        }
+    }
+    public static void main(String args[]) {
+        String greetings = "____________________________________________________________\n" +
+                " Hello! I'm Jeff\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n";
```

Comment:
When wrapping lines, break should be before an operator.
I noticed this issues in several other places as well. 
Do take note!
```suggestion
        String greetings = "____________________________________________________________\n" 
                + " Hello! I'm Jeff\n"
                + " What can I do for you?\n"
                + "____________________________________________________________\n";
```